src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -63,11 +86,123 @@ const NavMenu = (): ReactElement => {
+              anchorEl={anchorRef.current}
+              role={undefined}
+              transition
+              disablePortal
+            >
+              {({ TransitionProps, placement }) => (
+                <Grow
+                  {...TransitionProps}
+                  style={{
+                    transformOrigin:
+                      placement === 'bottom' ? 'center top' : 'center bottom',
+                  }}
+                >
+                  <Paper>
+                    <ClickAwayListener onClickAway={handleReportsMenuClose}>
+                      <MenuList
+                        autoFocusItem={reportsMenuOpen}
+                        id="menu-list-grow"
+                      >
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/donations"
+                          as={`/accountLists/${state.accountListId}/reports/donations`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText primary={t('Donations')} />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/partner"
+                          as={`/accountLists/${state.accountListId}/reports/partner`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText
+                              primary={t('14-Month Report (Partner)')}
+                            />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/salary"
+                          as={`/accountLists/${state.accountListId}/reports/salary`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText
+                              primary={t('14-Month Report (Salary)')}
+                            />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/designation_accounts"
+                          as={`/accountLists/${state.accountListId}/reports/designation_accounts`}
+                          scroll={false}
+                        >
+                          <MenuItem onClick={handleReportsMenuClose}>
+                            <ListItemText primary={t('Designation Accounts')} />
+                          </MenuItem>
+                        </NextLink>
+                        <NextLink
+                          href="/accountLists/[accountListId]/reports/financial_accounts"
```

Comment:
Let's make the url match what the UI is actually calling the page. I think there's been some iteration and renaming here.